core-bundle/tests/EventListener/DataContainer/RecordPreviewListenerTest.php

```diff
@@ -29,22 +31,39 @@ public static function tearDownAfterClass(): void
         unset($GLOBALS['TL_DCA']);
     }
 
+    protected function tearDown(): void
+    {
+        unset($GLOBALS['TL_DCA']);
```

Comment:
This line might habe killed the test suite (e.g. `PreviewLinkListenerTest`) in php 8.1 and 8.2.

and a call to `parent::tearDown()` is missing.

Reply:
To clarify: I think `unset($GLOBALS['TL_DCA']);` is correct here, but probable some other tests have to be fixed now 🙈

Reply:
NOW this line is not needed anymore ^^ see e85e216